test/memspaces/memspace_highest_bandwidth.cpp

```diff
@@ -9,16 +9,20 @@
 #include "memspace_internal.h"
 #include "test_helpers.h"
 
-static bool canQueryBandwidth(size_t nodeId) {
+static void canQueryBandwidth(size_t nodeId) {
     hwloc_topology_t topology = nullptr;
     int ret = hwloc_topology_init(&topology);
-    UT_ASSERTeq(ret, 0);
+
```

Comment:
are these extra empty lines needed? (added in all places in this PR)

Reply:
No, not needed. I have probably added them on "aesthetic" purpose, I'll just delete them.